src/client/features/search/index.js

```diff
@@ -63,12 +63,10 @@ class Search extends React.Component {
   getLandingResult() {
     const state = this.state;
     const query = {
-      q: state.query.q.trim(),
-      type: 'ProteinReference',
-      datasource: state.query.datasource,
-      species: '9606'
+      genes: state.query.q.trim(),
+      target: 'UNIPROTSWISSPROT',
```

Comment:
We should probably make all the strings like these defined as enum-like objects in a central file.  Maybe not in this PR, but it should probably be done in a different future PR.

Reply:
What is 'UNIPROTSWISSPROT' and why - there is no such standard name... Why not simply use well-known 'UNIPROT' or any other official synonym for the preferred db name "UniProt Knowledgebase" according to the MIRIAM and Identifiers.org [record](https://www.ebi.ac.uk/miriam/main/datatypes/MIR:00000005)? Why everyone every time tends to invent a new name for known and standard things?

Reply:
UNIPROTSWISSPROT comes from g:convert, they have several different Uniprot target databases so gave each a different name.

Reply:
I knew this would come up :).

This is related to #524. Can you and @wxli0 coordinate to establish the set of 'target' biological database names?

Reply:
The set I'm using is https://github.com/PathwayCommons/app-ui/pull/532/commits/b75645d11ee4c90771060575314cf8fdbcbf4606#diff-a10c2f2341a355021b49217584c19d14R256. Which ones need to be changed?

Reply:
Let's stick to the following set of names, case-insensitive (g:Convert name in parentheses):

1)  'ENSG' ('ENSG') 
2 ) 'HGNC' ('HGNC') 
3)  'HGNC_ID' ('HGNC_ACC')
4)  'UNIPROT' ('UNIPROTSWISSPROT')
5)  'ENTREZGENE' ('ENTREZGENE_ACC')

Reply:
Are 'ENTREZGENE' and 'ENTREZGENE_ACC' the same database?
https://github.com/PathwayCommons/app-ui/issues/524#issuecomment-373731310

Reply:
Please, let’s use official db names (ID types) from identifiers.org.
Instead of ‘entrezgene’ (obsolete) or alike, use ‘NCBI Gene’ (e.g., 1234); by ‘HGNC’ we probably here mean ‘HGNC Symbol’ (aka gene name); ‘HGNC’ points to IDs like HGNC:12345 (and these are not widely used nor very useful...)

Reply:
see #524

Reply:
Just to summarize #524:

| validator parameter | g:Convert parameter name | 
|-------------------- |----------------------|
| ENSG                       | ENSG      | 
| HGNCSYMBOL      | HGNC      |
| HGNC                       | HGNC_ACC    |
| UNIPROT                       | UNIPROTSWISSPROT      |
| NCBIGENE                       | ENTREZGENE_ACC      |